flo-runner/src/main/java/com/spotify/flo/context/Logging.java

```diff
@@ -101,4 +123,17 @@ void printPlan(TaskInfo taskInfo) {
     PrintUtils.tree(taskInfo).forEach(LOG::info);
     LOG.info("");
   }
+
+  @AutoValue
+  static abstract class Status {
+    abstract boolean success();
+    abstract Optional<Throwable> failure();
```

Comment:
Could we just have absence of failure mean success? So "Unknown Error" printout is prevented by design.

Reply:
👍